NBitcoin.TestFramework/WellknownNodeDownloadData.cs

```diff
@@ -1,12 +1,37 @@
 using System;
+using System.Linq;
 
 namespace NBitcoin.Tests
 {
 	public partial class NodeDownloadData
 	{
 		public bool SupportCookieFile { get; set; } = true;
 
-		public class BitcoinNodeDownloadData
+		public class NodeDownloadDataBase

```

Comment:
this seems unrelated to the PR, pushed by mistake?

Reply:
Unrelated to this PR, I did a separate commit.

Reply:
but the NodeDownload commit had signing-related things too xD